src/software/ai/hl/stp/tactic/attacker/attacker_fsm.cpp

```diff
@@ -11,10 +13,13 @@ void AttackerFSM::pivotKick(const Update& event,
     }
     // default to chipping the ball away
     PivotKickFSM::ControlParams control_params{
-        .kick_origin       = ball_position,
-        .kick_direction    = (chip_target - ball_position).orientation(),
-        .auto_chip_or_kick = AutoChipOrKick{AutoChipOrKickMode::AUTOCHIP,
-                                            (chip_target - ball_position).length()}};
+        .kick_origin    = ball_position,
+        .kick_direction = (chip_target - ball_position).orientation(),
+        .auto_chip_or_kick =
+            AutoChipOrKick{AutoChipOrKickMode::AUTOCHIP,
+                           (chip_target - ball_position)
+                               .length()}};  // TODO (NIMA): We should not randomly chip
+                                             // the ball away, specially towards enemy net
```

Comment:
towards the enemy net might make sense. if we're stuck, it can help us get the ball away to the enemy side and get possession there